api/v1alpha1/loadbalancer_types.go

```diff
@@ -134,6 +144,45 @@ type Cookie struct {
 	Attributes map[string]string `json:"attributes,omitempty"`
 }
 
+// ClientSideWeightedRoundRobin defines configuration for Envoy's Client-Side Weighted Round Robin policy.
+// See Envoy proto: envoy.extensions.load_balancing_policies.client_side_weighted_round_robin.v3.ClientSideWeightedRoundRobin
+// Note: SlowStart is not supported for this policy in Envoy Gateway at this time.
+type ClientSideWeightedRoundRobin struct {
+	// Whether to enable out-of-band utilization reporting collection from the endpoints.
+	// By default, per-request utilization reporting is used.
+	// +optional
+	EnableOOBLoadReport *bool `json:"-"`
+
+	// Load reporting interval to request from the server. Used only when enableOOBLoadReport is true.
+	// Default is 10s; server may not provide reports as frequently as requested.
+	// +optional
+	OOBReportingPeriod *gwapiv1.Duration `json:"-"`
```

Comment:
If you're keeping these fields in (ref https://github.com/envoyproxy/gateway/pull/7407#discussion_r2487875619) you should include a `notImplementedHide` tag.

```suggestion
	// Whether to enable out-of-band utilization reporting collection from the endpoints.
	// By default, per-request utilization reporting is used.
	//
	// +optional
	// +notImplementedHide
	EnableOOBLoadReport *bool `json:"-"`

	// Load reporting interval to request from the server. Used only when enableOOBLoadReport is true.
	// Default is 10s; server may not provide reports as frequently as requested.
    //
	// +optional
	// +notImplementedHide
	OOBReportingPeriod *gwapiv1.Duration `json:"-"`
```

Reply:
Sure so do I still need to make the json as - with this tag? And do I keep it or remove it completely?

Reply:
I have removed the fields above as I see no plans to implement this in envoy as of now